modules/ROOT/partials/plugin-files/plugin-file-list-advcode.adoc

```diff
@@ -1,6 +1,19 @@
-....
+.Base js file
+[source, js]
+----
+./plugins/advcode/plugin.js
+----
+
+.css
+[source, js]
+----
 ./plugins/advcode/codemirror.min.css
 ./plugins/advcode/codemirror.min.js
 ./plugins/advcode/customeditor.js
```

Comment:
I think these need it's own section like `lazy loaded files` or `extra files` it should not be under the css section since these are not css related.

Reply:
Separated all instances of this into there own sections labeled as `.other js files` as discussed.